docs/project/2024/roman.gulida/index.md

```diff
@@ -0,0 +1,97 @@
+The web server running on the Pico WH allows users to access the tracker remotely through a web page, providing a convenient interface for monitoring prices and configuring alerts.
+
+## Log
+
+
+<!-- write every week your progress here -->
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+In my project I used a 1.8-inch SPI LCD with the ST7735 controller (128 x 160 px) to display cryptocurrency prices and daily percentage changes. The central microcontroller is the Raspberry Pi Pico WH. I used a breadboard for prototyping, a 3V active buzzer for auditory price alerts, toggle switch for switching between currencies and jumper wires to connect everything.
+
+### Schematics
+
+Place your KiCAD schematics here.
+
+### Bill of Materials
+
+<!-- Fill out this table with all the hardware components that you might need.
+
+The format is 
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico WH](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [39 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12395-raspberry-pi-pico-wh.html?search_query=raspberry+pi+pico+WH&results=34) |
+| [LCD Display](https://www.displayfuture.com/Display/datasheet/controller/ST7735.pdf) | Display price and daily change | [30 RON](https://www.optimusdigital.ro/en/lcds/3554-modul-lcd-de-18-cu-spi-i-controller-st7735-128x160-px.html?search_query=ST7735&results=13) |
+| [Active Buzzer](https://www.mouser.com/datasheet/2/400/ef532_ps-13444.pdf) | Price alert | [1.5 RON](https://www.optimusdigital.ro/en/buzzers/635-3v-active-buzzer.html?search_query=active+buzzer&results=16) |
+| [Breadboard](https://busboard.com/documents/datasheets/BPS-DAT-(KIT-BB830+SB830)) | The physical base of the project | [10 RON](https://www.optimusdigital.ro/en/breadboards/8-breadboard-hq-830-points.html) |
+| [Jumper Wires](https://ro.mouser.com/datasheet/2/58/BPS_DAT__ZW_10__Datasheet-1282929.pdf) | Connect components | [4 RON](https://www.optimusdigital.ro/en/wires-with-connectors/887-set-fire-tata-tata-10p-15-cm.html?search_query=jumper+wires&results=100) |
+| [Toggle Switch]() | Switch between cryptocurrencies | [0.5 RON](https://www.optimusdigital.ro/en/buttons-and-switches/8930-miniature-side-toggle-switch-with-5-mm-handle.html?search_query=toggle+switch&results=31) |
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7735](https://github.com/scholtzan/st7735-rs) | Display driver for ST7735 | Used for the display for the Pico WH |
+| [reqwest](https://github.com/seanmonstar/reqwest) | HTTP Client | Retrieve cryptocurrency prices and other relevant data from a remote server |
```

Comment:
I don't think this is `no_std` and I think it does not work on the Raspberry Pi Pico.